controllers/pwa.js

```diff
@@ -49,7 +53,9 @@ router.get('/', (req, res, next) => {
         sortOrder: sortOrder,
         showNewest: sortOrder !== 'newest',
         showScore: sortOrder !== 'score'
-      });
+      };
+      libMetadata.add(arg, req);
```

Comment:
I'm not sure, but it might be cleaner if this is flipped around, so that the default arguments are retrieved first, and are then overwritten with specifics. So it would be something like:

```javascript
let arg = Object.assign(libMetadata.fromRequest(req), {
  title: 'PWA Directory',
  description: '...'
});
res.render('pwas/list.hbs', arg);
```

This would also eliminate the `addMissingMetadata()` method from `metadata.js`. It also wouldn't be necessary to pass `arg` to `libMetadata.add()`.